MaxText/configs/base.yml

```diff
@@ -44,6 +44,7 @@ load_full_state_path: ""
 # async_checkpointing is true, else a synchronous one is used. If you have
 # problems with the checkpointer we recommend trying the synchronous one.
 enable_checkpointing: True
+save_checkpoint_on_completion: True
```

Comment:
Has this been the default historically? I am wondering if it would be worthwhile to set the default to False. It seems to slow down a lot of testing having this enabled